src/models/session/logic.ts

```diff
@@ -107,41 +107,6 @@ export function sortBreadcrumbs(
   return orderedBreadcrumbs;
 }
 
-export function findNextNodeOfType({
```

Comment:
This function (and associated tests) has been removed as it's not used, and doesn't work as expected.